developers/weaviate/starter-guides/custom-vectors.mdx

```diff
@@ -8,187 +8,207 @@ image: og/docs/tutorials.jpg
+  |  7 | SCIENCE    | In 1953 Watson & Crick built a model of the molecular structure of this, the gene-carrying substance              | DNA                     | [ -0.014227471, 0.020493254, ..., -0.0027445166 ]  |
+  |  8 | SCIENCE    | Changes in the tropospheric layer of this are what gives us weather                                               | the atmosphere          | [ 0.009625228, 0.027518686, ..., -0.0068922946 ]   |
+  |  9 | SCIENCE    | In 70-degree air, a plane traveling at about 1,130 feet per second breaks it                                      | Sound barrier           | [ -0.0013459147, 0.0018580769, ..., -0.033439033 ] |
 
-### Login to Weaviate Cloud
+</details>
 
-1. Open the [WCD login page](https://console.weaviate.cloud/) in a browser.
-1. Click `Login to Weaviate Cloud Services`.
-1. Enter your email address and password to authenticate.
+### Weaviate Instance
 
-### Create a sandbox cluster
+Weaviate is open source. You can [run Weaviate](/developers/weaviate/installation) locally, in the cloud, or as a service. If you want to follow this guide without setting up an instance of your own, consider using a free, Sandbox instance in [Weaviate Cloud]/developers/wcs/quickstart).
 
-To create a cluster, click the 'Create cluster' button on the WCD Dashboard page.
+### Client library
 
-import CreateClusterButton from '/developers/wcs/img/wcs-create-cluster-button.jpg';
+[Client libraries](/developers/weaviate/client-libraries) simplify working with Weaviate. Clients are available for multiple programming languages. This guide provides examples in Python, Typescript, and cURL.
 
-<center><img src={CreateClusterButton} width="75%" alt="Create cluster button"/></center>
+To install a client library, use the installer for the client language:
 
-import CreateSandboxCluster from '/_includes/wcs/create-cluster-sandbox.mdx'
+<Tabs groupId="languages">
+<TabItem value="py" label="Python Client v4">
+The v4 client requires Weaviate 1.23 or higher.<br/><br/>
 
-<CreateSandboxCluster />
+```bash
+pip install -U weaviate-client
+```
+</TabItem>
+<TabItem value="py3" label="Python Client v3">
 
-## Connect to your sandbox
+```bash
+pip install "weaviate-client==3.*"
+```
+</TabItem>
+<TabItem value="js" label="JS/TS Client v3">
 
-Follow these steps to connect to your sandbox instance.
+```bash
+npm install weaviate-client --tag next
+```
+</TabItem>
+<TabItem value="js2" label="JS/TS Client v2">
 
-### Get the connection details
+```bash
+npm install weaviate-ts-client
+```
+</TabItem>
+</Tabs>
 
-To connect to your sandbox instance, you need the following information:
+## Connect to Weaviate
 
-- The Weaviate sandbox URL
-- The Weaviate sandbox API key
+To connect to your Weaviate instance, you need the following information:
```

Comment:
I think this is only true for Weaviate Cloud instances, or if the user had explicitly set up authentication for their local instance. 

Perhaps we could just commit a verbiage that follows using a Weaviate cloud instance to avoid a series of conditional statements re: authentication.